containeranalysis/setup.py

```diff
@@ -23,7 +23,7 @@
 name = "google-cloud-containeranalysis"
 description = "Container Analysis API API client library"
 version = "0.1.0"
-release_status = "3 - Alpha"
+release_status = "Development Status :: 3 - Alpha"
```

Comment:
Do we need to fix a template somewhere for this error?

Reply:
Looks like it, I think the one in GAPIC Generator always sets the release_status to a bad classifier.